indico/modules/events/registration/forms.py

```diff
@@ -610,6 +611,33 @@ def validate_retention_period(self, field):
                                     'that of individual fields.'))
 
 
+class RegistrationBasePriceForm(IndicoForm):
+    action = SelectField(_('Action'), [DataRequired()])
+    base_price = DecimalField(_('Registration fee'),
+                              [NumberRange(min=0, max=999999.99), HiddenUnless('action', 'custom'), DataRequired()],
```

Comment:
Shouldn't this be `min=0.01`? When submitting with `0` it currently fails the `DataRequired` validator which is still better than not having a validation error, but the message is not as nice as it'd be when failing the `NumberRange` validator.

Reply:
I keep getting this error when setting `min=0.01` tho...

![image](https://github.com/indico/indico/assets/27357203/43dacf2b-81b1-46d6-995b-c6dcd09636c8)

Reply:
it seems that something funky is going on with the `Decimal` type..

I tried adding these prints in the `NumberRange` validator:
```python
print('data', data)
print('min', self.min)
print(data >= self.min)
```
and this was the ouput:
```
data 0.01
min 0.01
False
```

and if I set `min=Decimal(0.01)` this happens:
![image](https://github.com/indico/indico/assets/27357203/60e53501-a380-403a-971d-b884266af048)

Reply:
`Decimal('0.01')`?

Reply:
🤦‍♂️@ myself
thanks @tomasr8 !